salt/modules/pip.py

```diff
@@ -688,6 +688,14 @@ def install(pkgs=None,  # pylint: disable=R0912,R0913,R0914
         cmd.append('--no-index')
 
     if mirrors:
+        # https://github.com/pypa/pip/pull/2641/files#diff-3ef137fb9ffdd400f117a565cd94c188L216
+        pip_version = version(pip_bin)
+        if salt.utils.compare_versions(ver1=pip_version, oper='>=', ver2='7.0.0'):
+            raise CommandExecutionError(
+                    'pip >= 7.0.0 does not support --use-mirrors argument: '
+                    'use --index-url and/or --extra-index-url instead'
+            )
```

Comment:
Can you change the error language here to use salt kwargs for this function instead of pip flags?